pyscriptjs/tests/integration/test_01_basic.py

```diff
@@ -276,3 +276,19 @@ def test_deprecated_globals(self):
             "Direct usage of sys is deprecated. Please use import sys instead.",
             "Direct usage of create is deprecated. Please use pyscript.create instead.",
         ]
+
+    def test_getPySrc(self):
```

Comment:
Small nit: what you think of this name change?

```suggestion
    def test_getPySrc_returns_code(self):
```

Reply:
Yeah, thanks, changed.